core/module/instance_methods_spec.rb

```diff
@@ -6,6 +6,11 @@
     methods = ModuleSpecs::Parent.instance_methods(false)
     methods.should_not include(:undefed_method)
   end
+  
+  it "does not consider methods as superclass methods if invoked on the parent module" do
```

Comment:
One important spec is missing here: `instance_methods(include_super=false)` includes only methods defined on that module.
Maybe there is a way to combine that and test when a module is included that the class `instance_methods(false)` does not change, as well as the module's.